### DISCUSSION TITLE
How to create a record on one table using a resource and create another record and save data on a different table using the same resource?

### DISCUSSION THREAD

**Discussion by otavio-araujo** [filamentphp/filament#2654] [Help]
How are you guys?

I have a table called Customer, and 2 others that recieve customer_id. One of them is to store data from a customer that is a person. The last one is used to store especific data from a customer tha is a company.

The tables:

1. customers;
2. pf_customers;
3. pj_customers.

The relationship is that customers hasMany pf_customer and hasMany pj_customer. A pf_customer bolongsTo a customer and a pj_customer belongsTo a customer.

Right.

How can create a Customer, and define the pf_customer data or the pj_customer data within the filament CustomerResouce?

Thank you guys!

**Comment by danharrin** [comment]
You can customise the save process? https://filamentphp.com/docs/2.x/admin/resources/creating-records#customizing-the-creation-process
